charts/gateway/values.yaml

```diff
@@ -30,6 +30,8 @@ imagePullSecrets: []
 # ```
 envFrom: []
 
+initContainers: {}
```

Comment:
Can you document it, ideally with an example. Consider adding the same example in tests, to make sure the chart renders correctly with this property set.

Reply:
Done! Let me know if you would like to see something changed.